Library/ViewModels/PledgeCTAContainerViewViewModel.swift

```diff
@@ -0,0 +1,92 @@
+import KsApi
+import Prelude
+import ReactiveExtensions
+import ReactiveSwift
+
+public protocol PledgeCTAContainerViewViewModelInputs {
+  func configureWith(project: Project, user: User)
+}
+
+public protocol PledgeCTAContainerViewViewModelOutputs {
+  var buttonBackgroundColor: Signal<UIColor, Never> { get }
+  var buttonTitleText: Signal<String, Never> { get }
+  var rewardTitle: Signal<String, Never> { get }
+  var stackViewIsHidden: Signal<Bool, Never> { get }
+}
+
+public protocol PledgeCTAContainerViewViewModelType {
+  var inputs: PledgeCTAContainerViewViewModelInputs { get }
+  var outputs: PledgeCTAContainerViewViewModelOutputs { get }
+}
+
+public final class PledgeCTAContainerViewViewModel: PledgeCTAContainerViewViewModelType,
+  PledgeCTAContainerViewViewModelInputs, PledgeCTAContainerViewViewModelOutputs {
+  public init() {
+    let projectAndUser = self.projectAndUserProperty.signal.skipNil()
+    let project = projectAndUser.map { $0.0 }
```

Comment:
We can be point free here: 
```suggestion
    let project = projectAndUser.map(first)
```